schemas/observer.yml

```
@@ -31,6 +31,18 @@
       type: keyword
       description: >
         Hostname of the observer.
+    - name: name
+      level: core
+      type: keyword
+      description: >
+        Name of the observer.
+      example: 1_proxySG
+    - name: product
+      level: core
+      type: keyword
+      description: >
+        The observers product name.
```

For consistency's sake with the other field definitions, please use

```suggestion
        The product name of the observer.
```

---

Another request, since we're going with @MikePaquette's s200 example, could you also add a corresponding example to the "vendor" field, with value "Symantec", please?